inst/table.m

```diff
@@ -424,19 +424,26 @@ function prettyprint (this)
     ## -*- texinfo -*-
     ## @node table.size
     ## @deftypefn {Method} {@var{sz} =} size (@var{obj})
+    ## @deftypefnx {Method} {[@var{nr}, @var{nv}] =} size (@var{obj})
```

Comment:
Need another line here for the N-argout case that indicates you can have _any_ number of argouts.

Reply:
Isn't a `table` by definition only 2-D ... rows x variables? Do we really need to support the N-argout case?

Reply:
Not quite: By definition, `table`s only have _nonzero_ length along dimensions 1 and 2, since they are 2-D arrays. And when you ask for `size(tbl)` you'll get a 2-long array back. And `ndims(tbl)` will always say "2".

But in Matlab and Octave, arrays of any dimensionality also, by definition, have an implicit number of 1-long trailing dimensions, which are only displayed or reported when you ask about them in certain ways, but nonetheless exist.

Consider the case of a scalar numeric:

```
>> x = 420.69;
>> size(x)
ans =
   1   1

>> size(x,3)
ans = 1
>> size(x,420)
ans = 1
>> [a,b,c,d,e,f,g,h] = size(x)
a = 1
b = 1
c = 1
d = 1
e = 1
f = 1
g = 1
h = 1
>>
```

Don't look at me, man; I don't make the rules, I just follow them. ;)